multi-agent/tests/unit/resources/conftest.py

```
@@ -0,0 +1,266 @@
+        for doc in self._docs.values():
+            if (
+                doc.identity.type == identity.type
+                and doc.identity.id == identity.id
+                and doc.category == category
+                and doc.type == type
+                and doc.name == name
+            ):
+                return doc.model_copy(deep=True)
+        return None
+
+    def find_resources(self, query: ResourceQuery) -> List[Resource]:
```

Ruff (0.16.0)

[warning] Unused method argument: `query`

(ARG002)